substrafl_examples/get_started/run_mnist_torch.py

```diff
@@ -451,7 +435,7 @@ def __init__(self):
 # - The :ref:`substrafl_doc/api/dependency:Dependency` to define the libraries on which the experiment needs to run.
 
 from substrafl.experiment import execute_experiment
-import time
+from substrafl.dependency import Dependency
```

Comment:
Since we no longer define `Dependency` in details beforehand, I would add a cell with a variation of 

```
# %%
# We also need to specify the third parties dependencies required to compute the metrics.
# The :ref:`substrafl_doc/api/dependency:Dependency` object is instantiated in order to install the right libraries in
# the Python environment of each organization.
```
as text, and the `algo_deps` definition in its own cell, before the big list recap. 

That way, the concept is introduced independently of `execute_experiment`

Reply:
I agree :)